README.md

```diff
@@ -163,6 +163,55 @@ helm tiller run \
 -    --set container.image=eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:latest
 ```
 
+## Visualising Benchmark Results
+
+You can use the `visualise_results.py` script to visualise benchmark results over time. 
+
+### Pre-Requisites
+
+You need to be authenticated with GCP in order to download the benchmark results. To do this use:
+
+`gcloud auth login --project <project_id>`
+
+`gcloud auth application-default login`
+
+Alternatively you can use the project service Account:
+
+- Go to the `eq-daily-performance-test` project in GCP
+- Go to IAM&Admin and open up the Service Accounts tab
+- Click on the `results-analysis` service account
+- Click edit, then, click 'Create Key'
+- Select JSON and store the file in a safe location - the path for this file will be used as an environment variable for the `get_benchmark_results.py` script.
+
+### Download Benchmark Results
+
+Run the `get_benchmark_results` script to download the latest benchmark results from GCP. You will need to set the bucket name as an environment variable:
+
+```
+GCS_OUTPUT_BUCKET - eq-daily-performance-test
+```
```

Comment:
If this was passed as a command line argument rather than an environment variable it would mean less documentation and it would be consistent with the visualise results script.

Reply:
I had it like this because it was already being used as an environment variable for the store_benchmark_outputs.py script. Perhaps I should update that one too?

Reply:
I think the get script should be consistent with the visualise script, and the store script left as is. It makes sense for store to use the environment variable as it's available in the container it runs in.